templates/deploy/params

```diff
@@ -0,0 +1,3 @@
+NAMESPACE=infographic-dev
+HTTPD_APPLICATION_NAME=httpd-app
+HTTPD_APPLICATION_HOSTNAME=
```

Comment:
Not sure we should leave this one blank - maybe just set it to some `example.com` value?